lib/stack_master/test_driver/cloud_formation.rb

```diff
@@ -106,7 +108,7 @@ def describe_change_set(options)
       def execute_change_set(options)
         change_set_id = options.fetch(:change_set_name)
         change_set = @change_sets.fetch(change_set_id)
-        update_stack(change_set)
+        @stacks[change_set.fetch(:stack_name)].attributes = change_set
```

Comment:
I've thought it too, but if this test CFn client mocks `update_stack` API, setting a stack policy is the original behavior.
So I think it's not a reasonable change to remove [this line](https://github.com/envato/stack_master/pull/178/files#diff-e733243b5c0c71cddbe1be6d3b39a442R159) and
either leaving the `update_stack` method as dead code for future use or deleting the method totally is an option.